src/theory/fp/type_enumerator.h

```diff
@@ -0,0 +1,128 @@
+/*********************                                                        */
+/*! \file type_enumerator.h
+ ** \verbatim
+ ** Original author: Morgan Deters
+ ** Major contributors: Martin Brain
+ ** Minor contributors (to current version): none
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2015  New York University and The University of Iowa
+ ** See the file COPYING in the top-level source directory for licensing
+ ** information.\endverbatim
+ **
+ ** \brief An enumerator for floating-point numbers.
```

Comment:
... and rounding modes?

Reply:
Seems like this file still has the old header format. Need to check if there are more of them.

Reply:
Yeah, it looks like this is also the case in the newest version of the branch.